audiofile/core/utils.py

```diff
@@ -76,7 +76,9 @@ def duration_in_seconds(
 
 def file_extension(path):
```

Comment:
Since it is not longer safe to call this function from outside the package, should we make this more explicit by renaming to `_file_extension()`?

Reply:
As none of the functions inside `utils.py` are part of the public API, I think it should be fine to stay with `file_extension()`. I also added a small comment, that we are not using `audeer.path()`.